ci-operator/step-registry/cluster-resource-override/s390x/run-tests/cluster-resource-override-s390x-run-tests-commands.sh

```
@@ -0,0 +1,23 @@
+echo "E2E_SKIP: ${E2E_SKIP:-<none>}"
+
+echo "=== Operator status before tests ==="
+oc get deployment,pods,csv,subscription -n "${CRO_NAMESPACE}" -o wide || true
+oc get clusterresourceoverride -A -o wide || true
+
+oc wait --for=condition=Available deployment/clusterresourceoverride-operator \
+  -n "${CRO_NAMESPACE}" --timeout=600s
+
+# make e2e expects OPERATOR_NAMESPACE and KUBECONFIG; KUBECONFIG is injected by ci-operator.
+export OPERATOR_NAMESPACE="${CRO_NAMESPACE}"
+export KUBECTL="$(which oc)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)